src/index.ts

```diff
@@ -1,13 +1,15 @@
 require('dotenv').config();
-import express from 'express';
+import { Telegraf, Context } from 'telegraf';
 
-const app = express();
-const port = process.env.PORT || 3000;
+const token = process.env.BOT_TOKEN;
 
-app.get('/', (_req, res) => {
-    res.send('Hello! I am event maker telegram bot');
-});
+if (!token)
+    throw new Error('Please assign a bot token to BOT_TOKEN variable in .env');
 
-app.listen(port, () => {
-    console.log(`Bot listening at http://localhost:${port}`);
-});
+const bot = new Telegraf(token);
```

Comment:
I propose to keep this logic in `App.ts` file and here just run something like
```
import { runApp } from './App.ts`
...
runApp(token)
```

Reply:
@fidelman I'd propose to put everything into `index.ts` until we feel it's worth refactoring. For the `hello world` initial implementation I think it's definitely ok.

Reply:
why wait if it is already clear that `index.ts` is the entry point of the application, not the core code. As well it creates unnecessary dependency bcz the next iteration will be moving this logic into a separated file.
Thi patter widely used even in telegraf https://github.com/telegraf/telegraf/blob/develop/src/index.ts

Reply:
@fidelman I know. I don't argue about this pattern or anything. I know how it's done. Many good developers (Kyle Simpson, Dan Abramov) are saying they don't care about having everything in `index.js` until it becomes huge and it's becomes hard to maintain it. I don't say we are going to put everything into `index.js`, but in our project I think it's fine to leave it like this for this commit for now, to move on faster. I will change it, it seems to be easier and faster.

Reply:
@bmind12 yeah, I totally get you. I am just thinking that the next iteration will definitely change this code.
So considering this application is not so big I'd like to try to avoid merge conflicts and I see here a potential one. Thanks

Reply:
@fidelman it's already split, please review